pkg/gatewayserver/grpc.go

```diff
@@ -28,16 +29,19 @@ func (gs *GatewayServer) GetGatewayConnectionStats(ctx context.Context, ids *ttn
 		return nil, err
 	}
 
+	uid := unique.ID(ctx, ids)
 	if gs.statsRegistry != nil {
 		stats, err := gs.statsRegistry.Get(ctx, *ids)
 		if err != nil || stats == nil {
```

Comment:
Should this be `&&`?

Reply:
No, but in case of `nil` I think the `errNotConnected` should be returned. `nil` implies not found here I believe.

Reply:
To be honest, the `err != nil` is enough here. We never return `nil` without an error, or an error without a `nil` response. Yet, we do this to be on the safe side and avoid returning a nil response.